Beis.LearningPlatform.Web/Views/Shared/Components/CmsSearchArticleListing/Default.cshtml

```diff
@@ -0,0 +1,91 @@
+﻿@model CmsSearchArticleListingViewModel
+<div class="landingpage-hero-list" id="@Model.Component.id">
+    <div class="govuk-width-container govuk-!-margin-bottom-6 govuk-!-margin-top-6 search-article-listing">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-three-quarters">
+                <div class="govuk-grid-row beis-columns-same-height">
+                    <div class="govuk-grid-column-full">
+                        <h2 class="govuk-heading-l govuk-!-padding-top-3 govuk-!-margin-bottom-2">@Model.Header</h2>
+                        <div class="top-line beis-yellow govuk-!-padding-top-2 govuk-!-margin-bottom-2"></div>
+                    </div>
+                </div>
+                @foreach (var article in Model.FullSearchArticles)
```

Comment:
Its worth breaking this section out to a Full search article component.

Reply:
Don't understand. It's called search but its more about filter tags

Reply:
Done. Please check